sycl/test-e2e/SYCLBIN/dg_executable.cpp

```diff
@@ -0,0 +1,28 @@
+//==----------- dg_executable.cpp --- SYCLBIN extension tests
+//-------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: aspect-usm_device_allocations
+
+// -- Test for using device globals in SYCLBIN.
+
+// Due to the regression in https://github.com/intel/llvm/issues/18432 it will
+// fail to build the SYCLBIN with nvptx targets. Once this is fixed,
```

Comment:
The linked issue is already fixed. Does it make sense to enable the test here, or would it be preferable to batch-enable all SYCLBIN tests (most of which seem to have this caveat) in a separate PR?